webpack.config.js

```diff
@@ -0,0 +1,84 @@
+const path = require("path")
+
+const webpack = require("webpack")
+const HtmlWebpackPlugin = require("html-webpack-plugin")
+
+const appDirectory = path.resolve(__dirname)
+const { presets, plugins } = require(`${appDirectory}/babel.config.js`)
+
+const compileNodeModules = [
+  // Add every react-native package that needs compiling
+  "react-native-gesture-handler",
+  "react-native-reanimated",
+].map((moduleName) => path.resolve(appDirectory, `node_modules/${moduleName}`))
+
+const babelLoaderConfiguration = {
+  test: /\.js$|tsx?$/,
+  // Add every directory that needs to be compiled by Babel during the build.
+  include: [
+    path.resolve(__dirname, "index.web.js"), // Entry to your application
+    path.resolve(__dirname, "app"),
+    ...compileNodeModules,
+  ],
+  use: {
+    loader: "babel-loader",
+    options: {
+      cacheDirectory: true,
+      presets,
+      plugins: ["react-native-web"],
+    },
+  },
+}
+
+const svgLoaderConfiguration = {
+  test: /\.svg$/,
+  use: [
+    {
+      loader: "@svgr/webpack",
+    },
+  ],
+}
+
+const imageLoaderConfiguration = {
+  test: /\.(gif|jpe?g|png)$/,
+  type: "asset/inline",
+}
+
+const ttfLoaderConfiguration = {
+  test: /\.ttf$/,
+  type: "asset/resource",
+}
```

Comment:
```suggestion
const ttfLoaderConfiguration = {
   test: /\.(woff(2)?|ttf|eot|svg)(\?v=\d+\.\d+\.\d+)?$/,
   use: [
      {
        loader: 'file-loader',
        options: {
          name: '[name].[ext]',
          outputPath: 'assets/fonts'
        }
      }
    ]
}
```

This article may be outdated, but it looks like you need to use `file-loader` here to get the fonts copied: https://chriscourses.com/blog/loading-fonts-webpack

Reply:
This did load them into the assets folder, if I inspect Sources via dev tools I do see the directory with the ttf files.

Still not rendering with the correct fonts, so maybe something with the css I was trying in the index.html 🤔